crates/cairo-lint-core/src/types.rs

```diff
@@ -0,0 +1,79 @@
+//Implement 'Felt252' struct with BigUint arithmetic operations
```

Comment:
remove that, if you need a felt type, use the one [here](https://github.com/starknet-io/types-rs/)